Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP1StateMachine.swift

```diff
@@ -469,6 +469,12 @@ extension HTTPConnectionPool {
             return jitteredBackoff
         }
     }
+
+    struct HTTP2MigrationContext {
+        var generator: Connection.ID.Generator
```

Comment:
Generator should be provided by the parent state machine.

Reply:
I removed it now completely, we no longer use it.